jupyterlab/staging/package.json

```diff
@@ -186,13 +186,13 @@
     "chokidar": "^3.4.0",
     "css-loader": "^6.7.1",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
-    "fs-extra": "^9.0.1",
+    "fs-extra": "^10.1.0",
```

Comment:
Let's discard these changes to staging as suggested by the CI check?

https://github.com/jupyterlab/jupyterlab/actions/runs/3487285189/jobs/5834734413